AppDashboard/lib/app_dashboard_data.py

```diff
@@ -647,3 +651,65 @@ def can_upload_apps(self):
     except Exception as err:
       logging.exception(err)
       return False
+
+  def get_dash_layout_settings(self, email=None):
+    """ Queries the UserAppServer to see what settings the user has saved
+    for customizing the UI of the Dashboard
+
+    Returns:
+      A JSON string describing the customization layout
+    """
+    default_dash_layout = '''
+        {"nav" :
+              {"App Management":{"True":
+                {"Upload App":"/apps/new",
+                "Delete App":"/apps/delete"}},
+              "AppScale Management":{"True":
+                {"Cloud Status":"/status/cloud",
+                "Manage Users":"/authorize"}},
+              "Debugging/Monitoring":{"True":
+                {"Monit":"temp1",
+                "TaskQueue":"temp2",
+                "Logging":"/logs"}}}
```

Comment:
Since you would like these to be ordered, I would recommend the following (Python code):
```
{'nav': [
    {'heading': 'App Management',
     'visible': True,  # Is this needed? Could the section just be removed from the structure before it's sent to the client?
     'buttons': {'Upload App': '/apps/new', 'Delete App': '/apps/delete'}},
    {'heading': 'AppScale Management',
     'visible': True,
     'buttons': {'Cloud Status': '/status/cloud'}},
    ...
]}
```